recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -136,6 +138,19 @@ pkg_postinst_ontarget_${PN} () {
 
 	# Ethernet RIO targets should not have RT startup settings
 	if ! [ "$TARGET_CLASS" = "Ethernet RIO" ]; then
+		HAS_CONSOLE=1
+		ARCH=$(uname -m 2>/dev/null)
+		if [ "$ARCH" = "x86_64" ]; then
+			efiConsoleOutEnable=$(/sbin/fw_printenv -n BootFirmwareConsoleOutEnable 2>/dev/null || true)
+			if [ -z "$efiConsoleOutEnable" ]; then
+				HAS_CONSOLE=0
+			fi
+		fi
+
+		if [ "$HAS_CONSOLE" = "1" ]; then
+			ln -sf ${settingsdatadir}/consoleout.ini ${systemsettingsdir}/consoleout.ini
```

Comment:
Since this is the only logic that we execute when `$HAS_CONSOLE` is true, can you just move it up into the `if [ -z "${efiConsoleOut...` block, and do-away with the `HAS_CONSOLE` indirection variable?